nanshe_workflow/data.py

```diff
@@ -118,9 +118,13 @@ def zip_zarr(name):
 
     io_remove(name_z)
 
-    with zarr.ZipStore(name_z, mode="w", compression=0, allowZip64=True) as f1:
+    # Opening in append mode to workaround a bug in `flush`.
+    # xref: https://github.com/alimanfoo/zarr/issues/158
+    with zarr.ZipStore(name_z, mode="a", compression=0, allowZip64=True) as f1:
```

Comment:
Opening in append mode to workaround a bug in `flush`. ( https://github.com/alimanfoo/zarr/issues/158 )